### PR TITLE
Add side titles text align #784

### PR DESCRIPTION
**Pull request by kzlakowski**
Hi, I've decided to take matters into my own hands and add feature requested in #784

### CODE REVIEW

**Comment by kzlakowski** [comment]
I've just realized there was a pull request for this - #730, but it seems to be no longer updated by it's creator.

File: lib/src/chart/base/axis_chart/axis_chart_data.dart [hunk 1]
```diff
@@ -357,6 +358,7 @@ class SideTitles with EquatableMixin {
     double? margin,
     double? interval,
     double? rotateAngle,
+    TextAlign? textAlign,
```
Comment (imaNNeoFighT) [inline lib/src/chart/base/axis_chart/axis_chart_data.dart:361]: Please put it below the `checkToShowTitle` property to keep the order correct.

File: lib/src/chart/base/axis_chart/axis_chart_data.dart [hunk 2]
```diff
@@ -326,6 +326,7 @@ class SideTitles with EquatableMixin {
   final double? interval;
   final double rotateAngle;
   final CheckToShowTitle checkToShowTitle;
+  final TextAlign textAlign;
```
Comment (imaNNeoFighT) [inline lib/src/chart/base/axis_chart/axis_chart_data.dart:329]: Or you can declare it above the `checkToShowTitle`. (everywhere you put it above `checkToShowTitle`)

**Comment by imaNNeoFighT** [comment]
Also please update the docs in `base_chart.md` file [here](https://github.com/imaNNeoFighT/fl_chart/blob/master/repo_files/documentations/base_chart.md#sidetitles).

**Comment by imaNNeoFighT** [comment]
Congratulations.
PR is approved!

Please fix those minor changes, and make sure to rebase with our `dev` branch.

File: lib/src/chart/base/axis_chart/axis_chart_data.dart (hunk 2 above)
Comment (kzlakowski) [reply lib/src/chart/base/axis_chart/axis_chart_data.dart:328]: Yeah, that's what I meant to do but missed that one, Good Catch :)

**Comment by imaNNeoFighT** [comment]
Thanks for contributing!
Your feature will be landed in the next version.

**Pull request merged**